benches/fs_open_close.rs

```diff
@@ -20,62 +17,56 @@ use rustposix::safeposix::cage::*;
 
 // Using this to include my criterion settings from a single shared file.
 // I did not use "use" or "mod" because benches/ isn't in the crate's usual
-// namespace and I didn't want to either make a separate crate with a single, 
+// namespace and I didn't want to either make a separate crate with a single,
 // tiny file or add this file to the rustposix crate.
 mod global_criterion_settings;
 
-
 pub fn run_benchmark(c: &mut Criterion) {
-
     // I'm following the initialization workflow from the unit tests here.
     //
-    // I'm using the lindrustinit to set up cages and the file system.  
+    // I'm using the lindrustinit to set up cages and the file system.
     rustposix::safeposix::dispatcher::lindrustinit(0);
 
     // Since all system calls are a method of a cage object, I also need this
-    // reference. 
+    // reference.
     let cage = interface::cagetable_getref(1);
 
-
-
     // --- COMPARING open / close CALLS ACROSS Lind + Native OS kernel ---
     let mut group = c.benchmark_group("Compare fs:open+close");
 
     // Should be similar.  Use a linear scale...
-    group.plot_config(criterion::PlotConfiguration::default().summary_scale(criterion::AxisScale::Linear));
+    group.plot_config(
+        criterion::PlotConfiguration::default().summary_scale(criterion::AxisScale::Linear),
+    );
 
     // Let's see how fast various file system calls are
-    group.bench_function("TF01: Lind open+close", |b| b.iter(||
-        {
-            let fd = cage.open_syscall("foo",O_CREAT | O_TRUNC | O_WRONLY,S_IRWXA);
+    group.bench_function("TF01: Lind open+close", |b| {
+        b.iter(|| {
+            let fd = cage.open_syscall("foo", O_CREAT | O_TRUNC | O_WRONLY, S_IRWXA);
             cage.close_syscall(fd);
-        }
-    ));
+        })
+    });
 
     // For comparison let's time the native OS...
-    group.bench_function("TF01: Native OS kernel open+close", |b| b.iter(||
-        {
-            unsafe{
-                let fd = libc::open(CString::new("/tmp/foo").unwrap().as_ptr(),O_CREAT | O_TRUNC | O_WRONLY,S_IRWXA);
-                libc::close(fd);
-            }
-        }
-    ));
+    group.bench_function("TF01: Native OS kernel open+close", |b| {
+        b.iter(|| unsafe {
+            let fd = libc::open(
+                CString::new("/tmp/foo").unwrap().as_ptr(),
+                O_CREAT | O_TRUNC | O_WRONLY,
+                S_IRWXA,
+            );
```

Comment:
this change here is a little weird.  It added a trailing "," on the last arg...

Reply:
I've [read more about this](https://doc.rust-lang.org/nightly/style-guide/#trailing-commas) and withdraw this point.  It's weird to me, but I guess it is Rustic...